### ISSUE TITLE
When i run a code inside  http://localhost:2358/dummy-client.html it gives me following error i am getting the following error

### ISSUE THREAD

**Issue by NischalKShaj** [judge0/judge0#492]
[Request 08/07/2024, 22:31:06] POST http://localhost:2358/submissions
{
    "source_code": "#include <stdio.h>\n\nint main(void) {\n  char name[10];\n  scanf(\"%s\", name);\n  printf(\"hello, %s\\n\", name);\n  return 0;\n}",
    "language_id": "50",
    "number_of_runs": null,
    "stdin": "Judge0",
    "expected_output": null,
    "cpu_time_limit": null,
    "cpu_extra_time": null,
    "wall_time_limit": null,
    "memory_limit": null,
    "stack_limit": null,
    "max_processes_and_or_threads": null,
    "enable_per_process_and_thread_time_limit": null,
    "enable_per_process_and_thread_memory_limit": null,
    "max_file_size": null,
    "enable_network": null
}
[Response 08/07/2024, 22:31:06] 201 Created
{
    "token": "d3589ee8-8d72-459b-8f8a-cd55fffb5fda"
}

[Request 08/07/2024, 22:31:08] GET http://localhost:2358/submissions/d3589ee8-8d72-459b-8f8a-cd55fffb5fda
[Response 08/07/2024, 22:31:08] 200 OK
{
    "stdout": null,
    "time": null,
    "memory": null,
    "stderr": null,
    "token": "d3589ee8-8d72-459b-8f8a-cd55fffb5fda",
    "compile_output": null,
    "message": "No such file or directory @ rb_sysopen - /box/main.c",
    "status": {
        "id": 13,
        "description": "Internal Error"
    }
}

[DONE 08/07/2024, 22:31:08]

**Comment by NischalKShaj** [comment]
> Same here, have you found any solutions for it?

Yeah in judge 0 many fields are selected as null or false by default change that to true and check

**Comment by narayanann** [comment]
i am facing the same issue as well , changing the value to true will help ?

{
    "stdout": null,
    "time": null,
    "memory": null,
    "stderr": null,
    "token": "7a32a49b-ccf6-4cbb-90de-8bd2b0d6bb83",
    "compile_output": null,
    "message": "No such file or directory @ rb_sysopen - /box/main.cpp",
    "status": {
        "id": 13,
        "description": "Internal Error"
    }
}

**Comment by Ignareo** [comment]
> i am facing the same issue as well , changing the value to true will help ?
> 
> { "stdout": null, "time": null, "memory": null, "stderr": null, "token": "7a32a49b-ccf6-4cbb-90de-8bd2b0d6bb83", "compile_output": null, "message": "No such file or directory @ rb_sysopen - /box/main.cpp", "status": { "id": 13, "description": "Internal Error" } }

check if your system using cgroupv2

I found in other issues that this Internal Error occurs when cgroupv1 is not able

her is my problem [I can't enable cgroupv1 in wsl2](https://stackoverflow.com/questions/79469388/how-to-enable-cgroup-v1-in-wsl2)

**Comment by NischalKShaj** [comment]
> i am facing the same issue as well , changing the value to true will help ?
> 
> { "stdout": null, "time": null, "memory": null, "stderr": null, "token": "7a32a49b-ccf6-4cbb-90de-8bd2b0d6bb83", "compile_output": null, "message": "No such file or directory @ rb_sysopen - /box/main.cpp", "status": { "id": 13, "description": "Internal Error" } }

 judge 0 dummy-client.html many fields are selected as null or false by default so chage that and test it in the dummy client whether it's working or not

**Comment by Ahingg** [comment]
> > i am facing the same issue as well , changing the value to true will help ?
> > { "stdout": null, "time": null, "memory": null, "stderr": null, "token": "7a32a49b-ccf6-4cbb-90de-8bd2b0d6bb83", "compile_output": null, "message": "No such file or directory @ rb_sysopen - /box/main.cpp", "status": { "id": 13, "description": "Internal Error" } }
> 
> check if your system using cgroupv2
> 
> I found in other issues that this Internal Error occurs when cgroupv1 is not able
> 
> her is my problem [I can't enable cgroupv1 in wsl2](https://stackoverflow.com/questions/79469388/how-to-enable-cgroup-v1-in-wsl2)

i faced the same issue on my ubuntu server 22.04. i'm using VMware workstation to run my ubuntu server and the problem has been solved by only changing GRUB_CMDLINE_LINUX="systemd.unified_cgroup_hierarchy=0" in /etc/default/grub. So if you only need local hosting for the judge0 i would suggest using VM instead of wsl. As from what i have searched, it's not really possible to change between v2 to v1.
